hw/ip/i2c/dv/env/i2c_env.core

```diff
@@ -10,6 +10,7 @@ filesets:
       - lowrisc:dv:cip_lib
       - lowrisc:dv:i2c_agent
       - lowrisc:dv:gen_ral_pkg
+      - lowrisc:ip:i2c
```

Comment:
Remove this - this is added to 
https://github.com/lowRISC/opentitan/blob/master/hw/ip/i2c/dv/i2c_sim.core

Reply:
Why is this error showing up now?

Reply:
could it be that you have `lowrisc:ip:i2c` dependencies in the env files below? if yes, this could lead to unstable builds, depending on other factors that influence the dependency resolution solver. i.e., in some builds, this might just work fine since (by chance) those dependencies where already included and compiled before this set. other builds might suddenly fail due to subtle changes in the dependency graph...

Reply:
@sriyerg: @imphil has placed the actual error messages inside the individual commit messages. could you PTAL?

Reply:
@tunghoang290780 
I would refrain from reusing the `i2c_reg_pkg` in DV code, since the same info is also encapsulated in the RAL model. Can we make do without this dependency?